quickstarts/oma-aqm/config.yml

```diff
@@ -0,0 +1,44 @@
+# Name of the quickstart (required)
+name: alert-quality-management
+
+# Displayed in the UI (required)
+title: Alert Quality Management Quickstart
+
+# Long-form description of the quickstart (required)
+description: |
+  Alert Quality Management (AQM) focuses on reducing the number of nuisance incidents so that you focus only on alerts with true business impact.  This reduces alert fatigue and ensures that you and your team focus your attention on the right places at the right times.
+  
+  For more information, see https://docs.newrelic.com/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide/
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Reduce the volume of nuisance incidents with the Alert Quality Management dashboard and process.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Alec Isaacson (New Relic)
+  - Andrew Faria (New Relic)
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - oma
+  - observabilty maturity
+  - AQM
+  - alert quality management
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - oma-aqm
```

Comment:
I didn't see any actual install plan in this directory for oma-aqm

https://github.com/newrelic/newrelic-quickstarts/tree/main/install

Reply:
You can read more about adding an install plan here: https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#installplans

Reply:
Removed installPlans: stanza.  The Fastly example that I followed also has an installPlans stanza that doesn't seem to point at an existing install plan.